src/lib.rs

```diff
@@ -99,6 +107,7 @@ impl Default for HomeserverConfig {
         HomeserverConfig {
             server_name: "localhost:9999".to_string(),
             public_baseurl: "http://localhost:9999".to_string(),
+            registration_shared_secret: HomeserverConfig::registration_default(),
```

Comment:
Nit: this function should probably be called `registration_shared_secret_default()`.